common/ux-window.cpp

```diff
@@ -742,9 +756,10 @@ namespace rs2
 
         ImGui::GetIO().MouseWheel = _mouse.ui_wheel;
         _mouse.ui_wheel = 0.f;
-
-        ImGui_ImplGlfw_NewFrame(_scale_factor);
-        //ImGui::NewFrame();
+        
+        ImGui_ImplOpenGL3_NewFrame();
```

Comment:
Please run the viewer and check memory consumption on task manager to make sure no memory leaks

Reply:
checked looks good